ngageoint/hootenanny-ui
# Translation assistant fix

- Added borders to TA mapping preset rows for more clarity between rows
- Fixed bug that caused mapping UI to render multiple times when re-uploading a translation

#### Testing instructions
With translation assistant panel open, upload a translation file (only .zip works at the moment). After upload is complete and the mapping UI has rendered, re-upload the same file. As a result, the mapping UI will be destroyed and re-rendered. _A duplicate should not be appended to the container_.
